roles/openshift_logging/tasks/patch_configmap_file.yaml

```diff
@@ -18,18 +18,24 @@
 - copy:
     content: "{{ __configmap_output.results.results[0]['data'][configmap_current_file] }}"
     dest: "{{ tempdir }}/current.yml"
+  when: configmap_current_file in __configmap_output.results.results[0]['data']
```

Comment:
will there always be a results[0]?

Reply:
yes because the configmap will exist in this task based on how it is called from `patch_configmap_files`